README.md

```diff
@@ -75,6 +75,37 @@ We recommend developers use Seq locally to help ensure the audit event developme
 
 ![Sample Seq Output](./assets/seq-events-view.png)
 
+## Using custom sink for large audit events
```

Comment:
I would moved this chapter above _Using Seq as the destination of audit events_ chapter

Reply:
Also move as a subchapter so use `###`

Reply:
Moved it above `Using Seq as..`. Regarding using `###`, the other sections like `Using Seq..` is using `##` hence I used this.